### PR TITLE
Include enrich into the info api as feature

### PR DESCRIPTION
**Pull request by martijnvg**
This commit also fixes a bug, the enrich enabled setting
was not included in the list of settings.

### CODE REVIEW

**Comment by martijnvg** [comment]
@elasticmachine run elasticsearch-ci/packaging-sample-matrix

File: x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichPlugin.java [hunk 1]
```diff
@@ -185,6 +190,7 @@ public EnrichPlugin(final Settings settings) {
     @Override
     public List<Setting<?>> getSettings() {
         return List.of(
+            ENRICH_ENABLED_SETTING,
```
Comment (hub-cap) [inline x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/EnrichPlugin.java]: doh!

**Review by hub-cap — APPROVED** [review]
awesome!

**Review by jbaiera — APPROVED** [review]
LGTM

**Pull request merged**